lib/reader/src/parser.rs

```diff
@@ -244,6 +244,13 @@ impl<'a> Context<'a> {
         }
     }
 
+    // Set the global for the stack limit.
+    fn set_stack_limit(&mut self, gv: GlobalVar) -> Result<()> {
+        // TODO: What do we do if there already is a stack_limit?
```

Comment:
Do we need to do something special here?

Reply:
In the parser here, we can report an error if there are multiple stack_limit declarations.